### DISCUSSION TITLE
etc/xiaoya中的配置共享文件

### DISCUSSION THREAD

**Discussion by cdok88** [monlor/docker-xiaoya#286] [General]
这个里面的.txt，包括夸克，ali盘的share.list好像copy过去，重启就会被删除掉。还有iptv.m3u放里面好像有的app不能解析，tivimate可以解析。

**Comment by monlor** [comment]
配置文件是持久化了的，修改alist容器的/data里面的内容就好了

**Reply by paramiao** [reply]
夸克 sharelist 怎么添加呢？只看到了 115 的 sharelist

**Reply by SimonLiXi** [reply]
直接把link加里面，然后选择要同步的文件夹和夸克挂载位置。
